synapse_data_warehouse/synapse/dynamic_tables/V2.33.4__file_latest_dynamic_table.sql

```diff
@@ -0,0 +1,22 @@
+-- Introduce the dynamic table
+USE SCHEMA {{database_name}}.synapse; --noqa: JJ01,PRS,TMP
+CREATE OR REPLACE DYNAMIC TABLE FILE_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS 
+    WITH dedup_filesnapshots AS (
+        SELECT
+            *
```

Comment:
You will need to explicitly name the columns here, so that when we add columns to the upstream table (SYNAPSE_RAW.FILESNAPSHOTS) the dynamic table doesn't break.

@jaymedina just made a PR to fix this for a dynamic table here: https://github.com/Sage-Bionetworks/snowflake/pull/119

Reply:
Thanks @philerooski . Another question I have is that I think it's better to rename the scripts to a later version instead of using v2.33.XXX since this is another attempt to update the table even though changes have already be pushed to dev database before. Does it sound good to you?

Reply:
@danlu1 Yes, good thinking. The revert wouldn't have dropped those records from the CHANGE_HISTORY table so you'll need to use a higher version.